omod/src/main/java/org/openmrs/module/providermanagement/rest/search/ProviderSearchHandler.java

```diff
@@ -0,0 +1,70 @@
+package org.openmrs.module.providermanagement.rest.search;
+
+import org.apache.commons.logging.Log;
```

Comment:
This is our recommendation for logging: https://openmrs.atlassian.net/wiki/spaces/docs/pages/25477044/Java+Conventions#[inlineExtension]Logging